### PR TITLE
fixed tests from feature 836

### PR DESCRIPTION
**Pull request by Isidor2811**

### CODE REVIEW

File: Animal.java [hunk 1]
```diff
@@ -1,8 +1,15 @@
 public class Animal {
 
-    public String name;
+    public String nname;
```
Comment (automator1aaa) [inline Animal.java]: Can you change this var name back to name, I think it was typo

File: Animal.java [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
     public int age;
     public double weight;
     public double height;
 
+    public void feedAnimal(){
+        System.out.println("Feeding...");
+    }
+
+    public void GoForAWalk(){
```
Comment (automator1aaa) [inline Animal.java]: Please check this method name (according to code convension)

File: TestClass.java [hunk 3]
```diff
@@ -11,22 +11,24 @@ public static void test(){
...
     }
 
     public void newMethod11(){
         //some code here
+        System.out.println(2);
     }
 
     public void bestMethod15(){
         //some code here
+        System.out.println(4);
```
Comment (automator1aaa) [inline TestClass.java:26]: Can you double check your code here. Maybe you need t0 sout 3 instead of 4

File: Animal.java (hunk 1 above)
Comment (Isidor2811) [reply Animal.java]: fixed

File: Animal.java (hunk 2 above)
Comment (Isidor2811) [reply Animal.java]: fixed

File: TestClass.java (hunk 3 above)
Comment (Isidor2811) [reply TestClass.java:26]: According to the test logic we have to print 4 here

**Review by automator1aaa — APPROVED** [review]
Approved

**Pull request merged**